drgn/helpers/linux/waitq.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2023, Oracle and/or its affiliates.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""
+Waitqueue
+--------------
+
+The ``drgn.helpers.linux.waitq`` module provides helpers for working with the
+wait-queue mechanism, implemented in :linux:`include/linux/wait.h`
```

Comment:
Since the header file is named `wait.h`, I think this file should be `wait.py` instead of `waitq.py`.

Reply:
Sure. Have renamed both the main and test files now.